### ISSUE TITLE
Look into physical data disposition requirements / process with Library and archives Canada

### ISSUE THREAD

**Issue by Br-Johnson** [dfo-pacific-science/data-stewardship-unit#123]
The issue of running out of physical storage space was brough uo by Marissa Ramey in the PSDCoP survey. They have so many boxes of data and they are running out of basement storage space. This is likely a common issue and it would be good to share documentation and best practices on what to do in this situation.

**Comment by stfinnis** [comment]
@Br-Johnson did you mean Marissa Glavas? Marisa Ramey isn't part of the PSDCoP. Or do you mean a different survey?
Either way, I'll start looking into this while you're away

**Comment by AnnaD-M** [comment]
@stfinnis yes, Brett would have meant Marissa Glavas, but Marisa Ramey with OCDS would be a good person to contact about compiling information/documentation about this.

**Comment by Br-Johnson** [comment]
Yes my mistake, Marissa Glavas.

**Comment by AnnaD-M** [comment]
Asked Marisa Ramey about this today as I emailing her about another topic.

**Comment by AnnaD-M** [comment]
Marisa responded that Information Management is responsible for setting guidelines for physical data disposition requirements, and provided this link: [https://intranet.ent.dfo-mpo.ca/imts-gist/en/node/1462](https://intranet.ent.dfo-mpo.ca/imts-gist/en/node/1462)
But she also mentioned that she had heard someone in the Maritimes (she thinks in Science) has been working on a Data Retention Policy (not information) and she will try to get the name for us.

**Comment by stfinnis** [comment]
This [info session related to the transfer of government records](https://catalogue.csps-efpc.gc.ca/product?catalog=DDN2-E35&cm_locale=en) came up in my emails. I think it's relevant, so I'll attend! It's on Feb 10, 2025
